<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JadePix</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="TabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Tab_Online">
       <attribute name="title">
        <string>Online Analysis</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Save Directory</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="LineEdit_Online_RegName">
              <property name="text">
               <string>//./xillybus_mem_8</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLineEdit" name="LineEdit_Online_FilePath">
                <property name="text">
                 <string>???</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Btn_Online_Choose">
                <property name="text">
                 <string>Choose..</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Register Path:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>File Name:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="LineEdit_Online_FileName">
              <property name="text">
               <string>test</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="LineEdit_Online_InName">
              <property name="text">
               <string>//./xillybus_read_32</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Device  Path:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>293</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <item row="0" column="1" colspan="2">
             <widget class="QSpinBox" name="SpinBox_Online_TimeRun">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="value">
               <number>60</number>
              </property>
              <property name="displayIntegerBase">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Time Run [ms]:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Events Display [mHz]:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="QSpinBox" name="SpinBox_Online_ChipAddress">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="displayIntegerBase">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Chip Address:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>NFiles:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QSpinBox" name="SpinBox_Online_NFiles">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="displayIntegerBase">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QSpinBox" name="SpinBox_Online_evDisplay">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="value">
               <number>1000</number>
              </property>
              <property name="displayIntegerBase">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>Row:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Column:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSpinBox" name="SpinBox_Online_SampleEvents">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
              <property name="value">
               <number>1000</number>
              </property>
              <property name="displayIntegerBase">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>Sample Events:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" alignment="Qt::AlignRight">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>1/</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1" colspan="2">
             <widget class="QSpinBox" name="SpinBox_Online_Col">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>47</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="displayIntegerBase">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="6" column="1" colspan="2">
             <widget class="QSpinBox" name="SpinBox_Online_Row">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>15</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="displayIntegerBase">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>ADC Threshold：</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1" colspan="2">
             <widget class="QSpinBox" name="SpinBox_Online_ADCThreshold">
              <property name="maximum">
               <number>65536</number>
              </property>
              <property name="value">
               <number>500</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_Online_Config">
            <property name="text">
             <string>Config</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>293</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Run control</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>State:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Label_Online_State">
                 <property name="text">
                  <string>???</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Frame Number:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Label_Online_FrameNumber">
                 <property name="text">
                  <string>???</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="Btn_Online_StartRun">
                 <property name="text">
                  <string>&amp;Start run</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Btn_Online_StopRun">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>S&amp;top run</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_9">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>Log</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QTextEdit" name="TextEdit_Online_Log">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCustomPlot" name="customPlot" native="true">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>600</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Action_Open"/>
    <addaction name="Action_Save"/>
    <addaction name="Action_Exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Action_Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="Action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="Action_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
